js/background.js

```diff
@@ -1355,22 +1399,22 @@ function isSafariContentBlockingAvailable() {
     return (SAFARI &&
             safari &&
             safari.extension &&
-            (typeof safari.extension.setContentBlocker === 'function'));
+            (typeof safari.extension.setContentBlocker === "function"));
 }
```

Comment:
function header format - are we using anon functions attached to variables or named functions?